src/libsystemd-network/sd-lldp.c

```diff
@@ -221,15 +215,14 @@ int lldp_handle_packet(tlv_packet *tlv, uint16_t length) {
                 case LLDP_TYPE_CHASSIS_ID:
 
                         if (len < 2) {
-                                log_lldp("Received malformed Chassis ID TLV len = %d. Dropping",
-                                         len);
+                                log_lldp("Received malformed Chassis ID TLV len = '%d'. Dropping.", len);
 
                                 malformed = true;
                                 goto out;
                         }
 
                         if (chassis_id) {
-                                log_lldp("Duplicate Chassis ID TLV found. Dropping ...");
+                                log_lldp("Duplicate Chassis ID TLV found. Dropping.");
```

Comment:
Shouldn't we give a little more context to the person possilby reading this log? I don't know the code very well, so I can't say what details deserve logging, but as it stands, it doesn't help much I fear.